composer.json

```diff
@@ -37,7 +37,7 @@
     "require-dev": {
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
         "sonata-project/user-bundle": "^3.6 || ^4.0",
-        "symfony/phpunit-bridge": "^5.0",
+        "symfony/phpunit-bridge": "^5.0 !=5.1.0",
```

Comment:
Any reason for not bumping to ^5.1.1 ?

Reply:
Just to not spread the scope of this PR more than required. We've found a problem that affects only 'symfony/phpunit-bridge:5.1.0", while there is no known problem at `5.0.x`.

Reply:
We are always using latest versions of phpunit bridge in sonata, there is no reason to have 5.0 there

Reply:
Yes, it's a dev dependency, we don't have to bother about this here.

Reply:
OK then. Constraint was updated.